src/util/psbt/mod.rs

```diff
@@ -116,7 +116,7 @@ impl<D: Decoder> Decodable<D> for PartiallySignedTransaction {
             return Err(Error::InvalidMagic.into());
         }
 
-        if 0xff_u8 != Decodable::consensus_decode(d)? {
+        if 0xff != u8::consensus_decode(d)? {
```

Comment:
Just wanted to mention that this somehow broke for me. It couldn't infer type, even though the 0xff_u8 is very explicitly typed. Same goes for the misc.rs change. Well, never hurts to be more specific I guess.